Orange/widgets/data/owselectrows.py

```diff
@@ -209,14 +207,14 @@ def __init__(self):
         self.cb_pc = gui.checkBox(
             box_setting, self, "purge_classes", "Remove unused classes",
             callback=self.conditions_changed)
-        layout.addWidget(box_setting, 0, 1)
+        layout.addWidget(box_setting, 0, 0)
 
         self.report_button.setFixedWidth(120)
         gui.rubber(self.buttonsArea.layout())
         layout.addWidget(self.buttonsArea, 1, 0)
 
         acbox = gui.auto_send(None, self, "auto_commit")
-        layout.addWidget(acbox, 1, 1)
+        layout.addWidget(acbox, 1, 0)
```

Comment:
It would be better to keep this on the right-hand side, e.g. `layout.addWidget(acbox, 0, 1)`. But also align it to the bottom. Perhaps replace the grid layout with a horizontal ... Ask me tomorrow and I'll help you.

Removing the title of the left box ("Purging") could also make sense.